pkg/manage/server.go

```diff
@@ -90,7 +90,7 @@ func ServerConfigFromEnv(loockup func(string) (string, bool)) *ServerConfig {
 
 // Addr return the address of the manage service.
 func (c *ServerConfig) Addr() string {
-	return c.Host + ":" + c.Port
+	return ":" + c.Port
```

Comment:
> The server must not listen on MANAGE_HOST. If so, no other service
can reach the container. I just removed the host form the address, maybe
this needs to be refactored later.

This I do not understand. In my full system test framework this setup works. It is just a question of correct network configuration in docker compose file I think.

Reply:
> docker compose

I had this error in docker swarm - the dns resolution works different there.